.github/workflows/release_draft.yml

```
@@ -67,13 +67,17 @@
             - { title: No entry 🚫, label: duplicate }
+            - { title: No entry 🚫, label: no RM }
             - { title: Breaking Changes 💥, label: breaking-change }
             - { title: New Features 🎉, label: enhancement }
             - { title: New Features 🎉, label: feature }
             - { title: Bugfixes 🛠, label: bug }
             - { title: Infrastructure ⚙️, label: infrastructure }
             - { title: Silent-live 🤫, label: silent-live }
             - { title: Documentation 📜, label: documentation }
+          coderabbit-support-active: 'true'
+          coderabbit-release-notes-title: 'Summary by CodeRabbit'
+          coderabbit-summary-ignore-types: [ 'Documentation', 'Tests', 'Chores', 'Bug Fixes', 'Refactor' ]
```

actionlint (1.7.7)

expected scalar node for string value but found sequence node with "!!seq" tag

(syntax-check)